### PR TITLE
fixing author metadata

### PR DESCRIPTION
**Pull request by jaybee84**
### Purpose

Fixing author metadata

#### Which areas should receive a particularly close look?

github handles for all authors

#### Is the pull request ready for review?

yes

### Manuscript checklist

NA

### CODE REVIEW

File: content/metadata.yaml [hunk 1]
```diff
@@ -5,47 +5,56 @@ keywords:
   - machine learning
   - transfer learning
 lang: en-US
-authors: # this is a randomised order of authors. The author order will be decided through discussion and mutual consent
+authors:  Below is a randomised order of authors. The author order will be decided through discussion and mutual consent
```
Comment (allaway) [inline content/metadata.yaml]: I think this needs to be commented out

**Review by allaway — APPROVED** [review]
I added some more details. and recommented-out the comment after "authors". feel free to merge if you agree with the changes!

**Review by allaway — APPROVED** [review]
I added some more details. and recommented-out the comment after "authors". feel free to merge if you agree with the changes!

**Review by cgreene — APPROVED** [review]

**Review by jaclyn-taroni — APPROVED** [review]
🚀

**Pull request merged**